File: atcoder/abc/089/B.py
from collections import Counter


def main():
    N = int(input())
    S = input().split()

    c = Counter(S)

    n_types = len(c.keys())
    if n_types == 3:
        print("Three")
    elif n_types == 4:
        print("Four")


if __name__ == '__main__':
    main()

File: atcoder/abc/089/B.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
  int N;
  cin >> N;

  string ans = "Three";
  for (int i = 0; i < N; i++) {
    string s;
    cin >> s;

    if (s == "Y") ans = "Four";
  }

  cout << ans << endl;

  return 0;
}
